QuadRelate.Tests/BoardHelperTests.cs

```diff
@@ -83,5 +83,59 @@ public void AvailableColumns_ForPartlyFullBoard_ReturnsFullList()
 
             Assert.Equal(expected, _board.AvailableColumns());
         }
+
+        [Fact]
+        public void IsGameOver_ForEmptyBoard_ReturnsFalse()
+        {
+            Assert.False(_board.IsGameOver());
+        }
+
+        [Fact]
+        public void IsGameOver_ForHorizontalBottomLeft_ReturnsTrue()
+        {
+            // Place counters in the first four columns of the bottom row
+            for (var x = 0; x < 4; x++)
+            {
+                _board[x, 0] = Cell.Red;
```

Comment:
In tests, you might be better off doing these as four lines:
E.g., 
_board[0, 0] = Cell.Red;
_board[1, 0] = Cell.Red; 
_board[2, 0] = Cell.Red; 
_board[3, 0] = Cell.Red; 

Production code gets more generic; tests get more specific.